contrib/test.bzl

```diff
@@ -76,13 +82,18 @@ _container_test = rule(
             values = [
                 "docker",
                 "tar",
+                "host",
```

Comment:
can you add to doc line above "Valid values are 'docker', 'tar', 'host'. nit: Also put these elements in alphabetical order.